src/components/UploadSong.tsx

```diff
@@ -59,11 +59,12 @@ const UploadSong: FunctionComponent<UploadSongProps> = ({
     [onChange, onBlur, onError]
   );
 
+  // TODO: Remove ".mp3" support after transcoded is fixed for supported files
   const { getRootProps, getInputProps, isDragActive } = useDropzone({
     onDrop: handleDrop,
     multiple: false,
     accept: {
-      audio: [".flac", ".fla", ".wav"],
+      audio: [".flac", ".fla", ".wav", ".mp3"],
```

Comment:
@scandycuz It seems like currently there's no support for transcoding the rest of the file types. So that we can test the upload flow i've added .mp3 files, we can delete after they add the support for it. What do you think?

Reply:
I'm not on review or called out here, but I just tested with .wav on local and the upload and generation of stream url worked. Not sure if this part is needed anymore.

The only issue I had was upload processing on the [PR preview link](https://newm-artist-portal--pr208-hotfix-upload-song-e-k7w1g421.web.app/), both mp3 and wav files didn't produce a stream url.

Reply:
>I'm not on review or called out here 
lol feedback is always welcome, i thought this was going to be merged before devops fixed the issue but they're freaky fast.

I was able to upload them fine locally... I will remove the mp3 as it is not longer needed